### DISCUSSION TITLE
Discord server?

### DISCUSSION THREAD

**Discussion by jrson83** [nanojsx/nano#62] [Ideas]
Hey yandeu,
are you maybe interessted in creating a Discord server for NanoJSX (or something similar)?
I'm really interessted in learning all the stuff but sometimes have a hard time following the examples.
It would be really cool if we build a small community on the library, maybe everyone could post some stuff like tutorials and help each other out.

**Comment by yandeu** [comment]
Feel free to create a server. I personally never use discord or similar.

**Reply by jrson83** [reply]
If anyone is interested joining the discord, here is the [link](https://discord.gg/xKSsSWVRdq). It would be cool if you can add this to the Nano JSX Github or Page somewhere. Well I'm alone in there lol.
